src/styles/utils/variables.scss

```diff
@@ -0,0 +1,7 @@
+// COLORS
+$header-hover-color: #00acdc;
+$white-accent: #FFF;
+$main-text-color: #060B35;
+$secondary-text-color: #616070;
+$blue-accent: #00ACDC;
```

Comment:
Why did you create the two variables with the same values?
<img width="245" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/47c6ef18-a296-4de6-ab72-27904d094fd2">
<img width="232" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/70232eaf-703a-4181-a2a1-c6b1494dcefb">
```suggestion
$white-accent: #FFF;
$main-text-color: #060B35;
$secondary-text-color: #616070;
$blue-accent: #00ACDC;
```